src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -527,16 +597,22 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
-            InputMedia thumb = default
+            InputMedia thumb = default,
+            bool allowSendingWithoutReply = default,
+            IEnumerable<MessageEntity> captionEntities = default,
+            bool disableContentTypeDetection = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.